### PR TITLE
Fixing /knimin/tests/test_ag_new_kit.py

### PR DESCRIPTION
**Pull request by josenavas**

### CODE REVIEW

**Comment by josenavas** [comment]
@sjanssen2 can you review this one? Pretty small and fast!

File: knimin/tests/test_ag_new_kit.py [hunk 1]
```diff
@@ -56,9 +56,12 @@ def test_post(self):
         tag = 'abc'
 
         # check for correct results
+        projects = db.getProjectNames()
+        project1 = projects[0]
+        project2 = projects[1]
         response = self.post('/ag_new_kit/',
                              {'tag': tag,
-                              'projects': ['PROJECT2', 'PROJECT5'],
+                              'projects': [project1, project2],
```
Comment (sjanssen2) [inline knimin/tests/test_ag_new_kit.py]: why do you change to lower letter project names?

File: knimin/handlers/ag_new_kit.py [hunk 2]
```diff
@@ -56,6 +55,6 @@ def post(self):
             kits = db.create_ag_kits(zip(num_swabs, num_kits), tag, projects)
             fields = ','.join(kits[0]._fields)
         except Exception as e:
-            raise HTTPError(500, "ERROR: %s" % str(e))
+            raise HTTPError(500, "ERROR: %s" % e.message.encode('utf-8'))
```
Comment (sjanssen2) [inline knimin/handlers/ag_new_kit.py]: cool. I did not know that you can get the text by ".message".

Comment (josenavas) [inline knimin/handlers/ag_new_kit.py]: 😄

File: knimin/tests/test_ag_new_kit.py (hunk 1 above)
Comment (josenavas) [inline knimin/tests/test_ag_new_kit.py]: note that what I did is creating two variables to store the name of two projects. Before they're strings 😃

Comment (sjanssen2) [inline knimin/tests/test_ag_new_kit.py]: stupid me, I really overlooked that!

**Comment by josenavas** [comment]
@sjanssen2 wanna merge?

**Comment by antgonza** [comment]
6 eyes, done ... but what if I was missing one eye?

**Pull request merged**